Jenkinsfile

```diff
@@ -155,24 +157,24 @@ pipeline {
           Run smoke tests for different servers and databases.
         */
         stage('Smoke Tests 01') {
-          agent { label 'linux && immutable' }
+          agent { kubernetes { yamlFile '.ci/k8s/OpenJdkPod.yml' } }
```

Comment:
IIRC, this was the way to ensure the parallel stages don't clash with the workspace when using the top-level k8s definition.

I need to clarify this to be sure this is the right pattern to be used.